deploy/helm/charts/templates/deployment.yaml

```
@@ -103,6 +103,10 @@ spec:
             - name: OPENEBS_IO_NFS_SERVER_NODE_AFFINITY
               value: "{{ .Values.nfsProvisioner.nfsServerNodeAffinity }}"
             {{- end }}
+            # Provide a switch to turn off the function of clearing stale pvc to avoid
+            # garbage collecting an NFS backend PVC if the NFS PVC is deleted.
+            - name: OPENEBS_IO_NFS_SERVER_GARBAGE_COLLECTION_ENABLED
+              value: "{{ .Values.nfsProvisioner.enableGarbageCollection }}"
```

nitpick:
```suggestion
            {{ if .Values.nfsProvisioner.enableGarbageCollection }}
            - name: OPENEBS_IO_NFS_SERVER_GARBAGE_COLLECTION_ENABLED
              value: {{ quote .Values.nfsProvisioner.enableGarbageCollection }}
            {{- end }}
```
And on the values.yaml, the option wouldn't need quotes around it anymore
```yaml
enableGarbageCollection: true
```